src/components/VerticalTabs/VerticalTabs.tsx

```diff
@@ -0,0 +1,206 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import {
+	findTypes,
+	getFirst,
+	omitProps,
+	Overwrite,
+	StandardProps,
+} from '../../util/component-types';
+import { buildModernHybridComponent } from '../../util/state-management';
+import * as reducers from './VerticalTabs.reducers';
+import {
+	IVerticalListMenuItemProps,
+	VerticalListMenuDumb as VerticalListMenu,
+} from '../VerticalListMenu/VerticalListMenu';
+
+const cx = lucidClassNames.bind('&-VerticalTabs');
+
+const { string, number, bool, func } = PropTypes;
+
+/** Vertical Tabs Tab Child Component */
+interface IVerticalTabsTabProps extends StandardProps {
+	/** Determines if the Tab is selected */
+	isSelected?: boolean;
+}
+
+const Tab = (_props: IVerticalTabsTabProps): null => null;
+
+Tab.displayName = 'VerticalTabs.Tab';
+Tab.peek = {
+	description: `
+        Content that will be rendered in a tab. Be sure to nest a Title
+        inside each Tab or provide it as a prop.
+    `,
+};
+Tab.propName = 'Tab';
+Tab.propTypes = {
+	isSelected: bool`
+        Determines if the Tab is selected.
+    `,
+};
+
+/** Vertical Tabs Title Child Component */
+interface IVerticalTabsTitleProps extends StandardProps {}
+
```

Comment:
Should we declare empty interfaces? 

We could just declare:
`const Title = (_props: StandardProps): null => null; `

This line produces an es-lint warning: 46:11  warning  An interface declaring no members is equivalent to its supertype  @typescript-eslint/no-empty-interface.

This rule aims to ensure that only meaningful interfaces are declared in the code. The error is discussed here:

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-empty-interface.md

We're not getting any type safety from the IVerticalTabsTitleProps interface, and, in any case, StandardProps only has optional types.
```
export interface StandardProps {
	/** Appended to the component-specific class names set on the root element.
			Value is run through the `classnames` library. */
	className?: string;
	/** Any valid React children. */
	children?: React.ReactNode;
	/** Styles that are passed through to native control. */
	style?: React.CSSProperties;
}
```

Reply:
I'm fine moving away from these empty interfaces. We never had a strong reason for keeping them other than simply getting a different name for the same thing.

Reply:
I removed the empty interface

Reply:
Great. I took it out.